### ISSUE TITLE
skyarch-networks___skyhopper yb analysis

### ISSUE THREAD

**Issue by keheliya** [project-kotinos/issues#195]
Add a yb configuration file to https://github.com/project-kotinos/skyarch-networks___skyhopper.
Instuctions are here: https://github.com/project-kotinos/root/blob/master/README.md 
If you come across problems, add them as comments to this issue.

**Comment by keheliya** [comment]
error:
```
+ bundle exec rake db:migrate:reset
bundler: failed to load command: rake (/workspace/vendor/bundle/ruby/2.4.0/bin/rake)
Gem::LoadError: You have already activated io-console 0.4.6, but your Gemfile requires io-console 0.4.7. Since io-console is a default gem, you can either remove your dependency on it or try updating to a newer version of bundler that supports io-console as a default gem.
```

**Comment by keheliya** [comment]
error: 
```
rake aborted!
ActiveRecord::AdapterNotSpecified: 'development' database is not configured. Available: ["test"]
```

**Comment by keheliya** [comment]
got it to work by setting `RAILS_ENV=test` similar to what travis does.
https://docs.travis-ci.com/user/environment-variables/#default-environment-variables
Fixed in skyarch-networks___skyhopper@03ff656deae6048fb2cd5a041859c466002bb4f6
